processing/src/main/java/io/druid/query/extraction/BucketExtractionFn.java

```diff
@@ -0,0 +1,109 @@
+package io.druid.query.extraction;
```

Comment:
yes

On Wed, Jun 15, 2016 at 3:51 PM, Dave Li <notifications@github.com> wrote:

> In
> processing/src/main/java/io/druid/query/extraction/BucketExtractionFn.java
> <https://github.com/druid-io/druid/pull/3033#discussion_r67260490>:
>
> > @@ -0,0 +1,109 @@
> > +package io.druid.query.extraction;
>
> does header refer to the "Licensed to Metamarkets Group..." comment?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/druid-io/druid/pull/3033/files/3da474e3eb58b835ae6341b3d2584bda959bc3d5#r67260490>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAaJJQK2qC-Okkj5vbXGUYZIXPaUNnIxks5qMIHxgaJpZM4Io3p9>
> .
>